contrib/node/examples/src/java/org/pantsbuild/testproject/jsresources/BUILD

```diff
@@ -0,0 +1,17 @@
+jvm_binary(name = 'jsresources',
```

Comment:
ack.  Is it recommended not to name "default" target explicitly?  I always names default targets so far.